src/FileSystem/FileSystem.cpp

```diff
@@ -642,7 +642,7 @@ unsigned long CFileSystem::getMBsFree(const std::string& path)
 {
 #ifdef _WIN32
 	ULARGE_INTEGER freespace;
-	BOOL res = GetDiskFreeSpaceEx(s2ws(path).c_str(), &freespace, nullptr, nullptr);
+	BOOL res = GetDiskFreeSpaceExW(s2ws(path).c_str(), &freespace, nullptr, nullptr);
```

Comment:
Is this sufficient? I assume, based on the name, that `s2ws` converts a string to a wide-string, so it should be sufficient, but just double checking.